stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -484,65 +519,89 @@ end
 #! Input: [pi_ptr, ...]
 #! Output: [norm_sq(s1), ...]
 #!
-#! Cycles: 58888
+#! Cycles: 40966
 export.compute_s1_norm_sq
     repeat.128
         # 1) Load the next 4 * 3 coefficients
-        # load c_i
+        # load the next four pi_i
         padw
-        dup.4 add.1281
+        dup.4
         mem_loadw
 
-        # load pi_{i+512}
+        # load the next four pi_{i+512}
         padw
         dup.8 add.128
         mem_loadw
 
-        # load pi_i
+        # load the next four c_i
         padw
-        dup.12
+        dup.12 add.1281
```

Comment:
Can we make 1281 a constant to ease readability (even if it's documented in the docstring)?

Reply:
Added